rebar.config

```diff
@@ -47,8 +47,9 @@
 
 {profiles,
  [{test,
-   [{deps, [{meck, {git, "https://github.com/eproxus/meck.git", {tag, "0.8.2"}}}]}]
+   [{deps, [{meck, {git, "https://github.com/eproxus/meck.git", {tag, "0.8.2"}}}]},
+    {erl_opts, [{d, 'LOG_TO_SYSLOG'}]}]
```

Comment:
Yeah the intent behind that line was to silence the logging of tests unless the case is failing in common test; Rebar3 can redirect the output of error_logger, lager, and `ct:pal`, but not of output done directly to `user` via batchio.